src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationService.java

```diff
@@ -333,6 +334,24 @@ private void getJurisdictionCodesErrors(List<String> errors, List<String> jurCod
         return errors;
     }
 
+    public void validateJudgementsHasJurisdiction(CaseData caseData, boolean partOfMMultiple, List<String> errors) {
+        if (caseData.getJudgementCollection() == null || caseData.getJudgementCollection().isEmpty()) {
```

Comment:
CollectionUtils.isNotEmpty covers both of these tests

Reply:
updated ^_^